model.py

```diff
@@ -586,6 +666,10 @@ def __init__(self):
         self._optimizer = None
         self._device = None
         self._device_ids = None
+        self._optimizer = None
+        self._distributed_sampler = None
```

Comment:
`self._distributed_sampler` seems unused elsewhere, should we remove it

Reply:
removed, thanks